charts/posit-chronicle/values.yaml

```diff
@@ -51,33 +69,88 @@ pod:
 # the chronicle server stateful-set
-storage:
-  persistentVolumeSize: 1Gi
 
+
+# Enable persistence using Persistent Volume Claims
+# ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
+#
+persistence:
+  # -- Enable persistence using Persistent Volume Claims
+  enabled: true
+  # -- Persistent Volume Storage Class
+  #    (Leave empty if using the default storage class)
+  storageClassName: ""
+  # -- Size of the data volume
+  size: 10Gi
+  # -- Persistent Volume Access Modes
+  accessModes:
+    - ReadWriteOnce
+  # -- Selector to match an existing Persistent Volume for the data PVC
+  selectorLabels: {}
+  # -- Additional annotations to add to the PVC
+  annotations: {}
+  # -- Additional labels to add to the PVC
+  labels: {}
+  # -- Finalizers added verbatim to the PVC
+  finalizers:
+    - kubernetes.io/pvc-protection
+
+# Additional secrets to mount to the Chronicle server pod
+extraSecretMounts: []
+# this option can be used to mount secrets such as an SSL certificate and key into the pod
+#  - name: "ssl"
+#    secretName: "chronicle-ssl"
+#    mountPath: "/etc/chronicle/ssl"
+#    optional: false
+#    items:
+#      - key: "tls.crt"
+#      - key: "tls.key"
+
+# Configurations for the underlying Chronicle server instance
+# ref: https://docs.posit.co/chronicle/appendix/library/advanced-server.html
+#
 config:
   HTTPS:
     # If https.enabled=true, ignore any http
     # values and enable https in the config instead
+    # -- If set to true, Chronicle will use HTTPS instead of HTTP
     Enabled: false
-    Key: ""
+    # -- Path to a PEM encoded TLS certificate file
     Certificate: ""
+    # -- Path to a PEM encoded private key file corresponding to the specified certificate
+    Key: ""
   Metrics:
-    Enabled: true
+    # -- If set to true, Chronicle will expose a metrics endpoint for Prometheus
+    Enabled: false
   Profiling:
+    # -- If set to true, Chronicle will expose a pprof profiling server
     Enabled: false
+    # -- The port to use for the profiling server
+    Port: 3030
   Logging:
+    # -- Specified the output for log messages, can be one of "STDOUT", "STDERR", or a file path
     ServiceLog: "STDOUT"
+    # -- The log level for the service, can be one of "TRACE", "DEBUG", "INFO", "WARN", or "ERROR"
     ServiceLogLevel: "INFO"
+    # -- The log format for the service, can be one of "TEXT" or "JSON"
     ServiceLogFormat: "TEXT"
+  # -- Configuration for local data storage with Chronicle, for configuring persistence of this data see the persistence section
   LocalStorage:
-    # By default LocalStorage.Enabled=true, so that installs work with the default values
+    # -- If set to true, Chronicle will use a local path for data storage. This should be used in conjunction with persistence.
     Enabled: true
-    Location: "./chronicle-data"
+    # -- The path to the local storage location
+    Path: "/opt/chronicle-data"
+    # -- The retention period for data before it is purged
```

Comment:
Should probably also remove RetentionPeriod from here.

Reply:
References to `RetentionPeriod` should now be removed.